src/extension.ts

```diff
@@ -191,16 +194,27 @@ function initCommandAsync(context: vscode.ExtensionContext,
         const properties: { [key: string]: string; } = {};
         properties.result = "Succeeded";
         TelemetryClient.sendEvent(`${commandId}.start`);
+        outputChannel.appendLine(`${commandId}: `);
         try {
             return await callback(...args);
         } catch (error) {
             if (error instanceof UserCancelledError) {
                 properties.result = "Cancelled";
+                outputChannel.appendLine(Constants.userCancelled);
             } else {
                 properties.result = "Failed";
-                errorData = new ErrorData(error);
-                outputChannel.appendLine(`Error: ${errorData.message}`);
-                vscode.window.showErrorMessage(errorData.message);
+                if (error instanceof LearnMoreError) {
+                    const learnMore: vscode.MessageItem = { title: Constants.learnMore };
+                    const items: vscode.MessageItem[] = [ learnMore ];
+                    outputChannel.appendLine(`Error: ${error.message}`);
+                    if (await vscode.window.showErrorMessage(error.message, ...items) === learnMore) {
+                        await opn(error.url);
```

Comment:
How about `vscode.open`? I saw that the Java debugger extension just switched from `opn` to `vscode.open` for unknown reasons: https://github.com/Microsoft/vscode-java-debug/blob/master/CHANGELOG.md#changed

Reply:
Thanks, will update